### PR TITLE
[ElasticSearch] Configure attribute filtering

### PR DESCRIPTION
**Pull request by Arminek**

### CODE REVIEW

File: src/Resources/config/app/config.yml [hunk 1]
```diff
@@ -25,18 +25,20 @@ ongr_filter_manager:
...
             document_field: ~
             request_field: page
             options:
                 count_per_page: 10
+        attributes:
+            type: multi_dynamic_aggregate
+            request_field: attributes
+            document_field: attributes>attributes.value
+            options:
+              name_field: attributes.name
```
Comment (lchrusciel) [inline src/Resources/config/app/config.yml]: Wrong indent

File: tests/Responses/Expected/product_list_page_by_en_gb_channel.json [hunk 2]
```diff
@@ -27,12 +27,10 @@
             ],
             "attributes":[
                 {
-                    "code":"MUG_COLLECTION_CODE",
```
Comment (lchrusciel) [inline tests/Responses/Expected/product_list_page_by_en_gb_channel.json]: Was it needed to remove this code from response? It breaks contract with shop api plugin.

File: README.md [hunk 3]
```diff
@@ -73,40 +73,42 @@ Elastic search for Sylius.
...
                     - search
+                    - attributes
                 repository: es.manager.default.product
         filters:
             channel:
                 type: choice
                 request_field: channel
                 document_field: channel_code
-            taxon_slug:
+            taxon_code:
```
Comment (lchrusciel) [inline README.md:88]: Shouldn't it be `taxonCode`? We can clean it later, but we should move to camelCase in all API responses.

File: README.md [hunk 4]
```diff
@@ -468,3 +470,667 @@ Response:
     }
 }
 ```
+
+10. Filtering by attributes:
+
+You need use attributes query parameter which is an associative array where key is the attribute name and value is an array of attribute values.
```
Comment (lchrusciel) [inline README.md:476]: need *to* use

Comment (lchrusciel) [reply README.md:476]: where *the* key is *an* attribute name and *the* value /cc @CoderMaggie

File: README.md [hunk 5]
```diff
@@ -468,3 +470,667 @@ Response:
...
+For e.g:
+```php
+$this->client->request('GET', '/shop-api/products', ['attributes' => ['Mug material' => ['Wood']]], [], ['ACCEPT' => 'application/json']);
```
Comment (lchrusciel) [inline README.md:479]: If I understand it correctly `Mug material` is an attribute code. If so, I would suggest to rename it to something more obvious like `MUG_MATERIAL_CODE`. Right now, it looks like attribute name.

File: README.md [hunk 6]
```diff
@@ -468,3 +470,667 @@ Response:
...
+              "label":"Wood",
+              "count":1
+            }
+          }
+        }
+      ]
+    }
+```
+
+You can combine filters so for example if you want your products to be filtered in specific locale you can add another query parameter
```
Comment (lchrusciel) [inline README.md:596]: Of course, you can combine filters. So, for example if you want

**Review by lchrusciel — APPROVED** [review]
README need one more iteration, but it can be done separately.

File: README.md (hunk 5 above)
Comment (Arminek) [reply README.md:479]: It is attribute name

Comment (lchrusciel) [reply README.md:479]: I don't have any idea, why the heck did I assume that it is a code. Fine then. Sorry, for false alarm :)

**Pull request merged**